### PR TITLE
Add a Recommendation File for Sydney, NSW

### LINKED ISSUE #4: sammo2828 recommendations for things to do in Sydney
I want to add a file describing fun things to do in Sydney

- [ ] Places to eat local flavours
- [ ] Fun things to do

### PR DESCRIPTION
**Pull request by sammo2828**
This closes #4 

Adding a file with recommendations for Sydney, NSW.

@nikosandronikos what should I add?

### CODE REVIEW

**Comment by delilah-cisra** [comment]
I think you're missing the culinary delights of Los Vida.

**Comment by sammo2828** [comment]
@delilah-cisra Done :sparkles:

File: sammo2828-sydney.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+## Sydney, NSW
+
+### Places to sample the local flavours
+
+* Los Vidda
```
Comment (delilah-cisra) [inline sammo2828-sydney.md]: s/Vidda/Vida :-)

**Comment by sammo2828** [comment]
@del i think this is ready for merging

**Review by delilah-cisra — APPROVED** [review]
Whatever:-)

**Pull request merged**